### PR TITLE
refactor: Replaced deprecated managedQuery() with getContentResolver().query()

### PR DESCRIPTION
**Pull request by CloudyPadmal**
@pa1pal I've removed my previous PR and made a new one to the _development_ branch! Can you review? :)

### CODE REVIEW

**Comment by CloudyPadmal** [comment]
With this fix, the features in the README can be improved right? :)

- [ ] **Upload** > Select Images which you want to upload [Doesn't crash now]
- [ ] **Upload** > Upload button which upload the image [Doesn't crash now]

File: app/src/main/java/vn/mbm/phimp/me/GalleryMap.java [hunk 1]
```diff
@@ -214,15 +214,21 @@ public void run()
...
-                        null,null,
-                        null);
-        		int columnIndex = cursor.getColumnIndexOrThrow(MediaStore.Images.Media.DATA);
-        		for(int i=0; i<cursor.getCount(); i++){
-        			if(cursor.moveToNext())
-        			filepath.add(cursor.getString(columnIndex));
-        		}	
+                        null,
+						null,
+                        null
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/GalleryMap.java]: Can you clear the integration, I know it's silly but it would be great if you could do so

Comment (CloudyPadmal) [inline app/src/main/java/vn/mbm/phimp/me/GalleryMap.java]: @Abhi2424shek there was some tabs and space mismatch in my IDE and now I've fixed it and made a PR! :D

**Comment by Abhi2424shek** [comment]
The diff looks fine, any screenshots?

**Comment by CloudyPadmal** [comment]
@Abhi2424shek I've added a screenshot! :D

**Comment by Abhi2424shek** [comment]
@CloudyPadmal sorry if my comment was not clear, I meant to have a screenshot in the PR body for reviewing. Anyways since it's good, pls remove that screenshot.

**Comment by CloudyPadmal** [comment]
@Abhi2424shek Ahh about the tabs and spaces right? :D I'll remove the screenshot! :+1:

**Comment by CloudyPadmal** [comment]
@Abhi2424shek Thank you!! :1st_place_medal:

**Pull request merged**